app/models/version_status.rb

```diff
@@ -35,6 +35,10 @@ def draft?
     status.open?
   end
 
+  def first_draft?
+    status.open? && status.version == 1
+  end
```

Comment:
Is this logic correct? I couldn't find any existing status that matched the situation where the initial draft is not yet deposited.

Reply:
I'd have to defer to @justinlittman. I'm not sure when we'd lean on the status vs. the deposit state machine.

Reply:
Yes, this logic is correct. Can you use this method in L23 to avoid dupe?

The deposit state machine has fairly specific uses. Whenever possible the version status should be used as it reflects the current status from SDR.